docs/devices.md

```diff
@@ -152,18 +144,71 @@ The `liveSlots` helper for constructing Vats provides both an `E()` wrapper
 (for constructing `syscall.send()` messages to Presences) and a `D()` wrapper
 (for constructing `syscall.callNow` messages to device nodes).
 
-### Device State Persistence
+### Device API
+
+Devices are very much like Vats, and they way they are constructed reflects
+that. The factory function for each is called with `syscall` object that
+they can use to talk to the kernel, and they return a `dispatch` object
+with which the kernel can invoke them. The specific methods on `syscall`
+and `dispatch` are different for devices:
+
+* `syscall.sendOnly(targetSlot, method, args) -> undefined`
+* `dispatch.invoke(deviceID, method, args) -> { results }`
```

Comment:
the return value is CapData, rather than an object with a `.results` property that contains capdata, so I'd remove the curlies

Reply:
done